deployments/engine/docker-compose/1m1e_with_s3.yaml

```diff
@@ -42,7 +42,6 @@ services:
       - "10241:10241"
     depends_on:
       - "server-master"
-    restart: "on-failure"
```

Comment:
Why not use `restart: unless-stopped` policy here?

Reply:
L49 has one.